include/boost/beast/core/detail/service_base.hpp

```diff
@@ -16,15 +16,10 @@ namespace boost {
 namespace beast {
 namespace detail {
 
-template<class T>
-struct service_id : net::execution_context::id
```

Comment:
Doesn't asio do it this way? I thought it was a workaround for a link problem... are you sure this change will be correct everywhere?

Reply:
It's possibly a workaround for a problem in a toolset that is not supported by beast anyway.